generator/src/pkg/generate/generate.go

```diff
@@ -97,33 +99,71 @@ func Parse(configFilename string) (model.FileConfig, []string) {
 	return loaded_config, clusters
 }
 
-func CreateK8sYaml(config model.FileConfig, clusters []string) {
+func CreateK8sYaml(config model.FileConfig, clusters []string, buildID int) {
```

Comment:
This function has gotten a bit too long... May be good to consider splitting into several ones for code readability?

Reply:
I agree, I will split the code that generates the templates out into a different function